### PR TITLE
Call a doctor

### PR DESCRIPTION
**Pull request by sobermoode**
Adding a story branch where you call a doctor or some assistance.

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/call-a-doctor/doctor.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+You take a deep breath, your chest puffing out against your restrictive jacket.
```
Comment (casey-collab) [inline english/call-a-doctor/doctor.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/call-a-doctor/ghostbusters.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+The door opens a crack and a doctor sticks his head in.
```
Comment (casey-collab) [inline english/call-a-doctor/ghostbusters.md]: This file has also been added to a directory that already contained a page.

File: english/call-a-doctor/hungry.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+"Dinner will be served at the standard time," the doctor explains.
```
Comment (casey-collab) [inline english/call-a-doctor/hungry.md]: This file has also been added to a directory that already contained a page.

File: english/call-a-doctor/let-me-out.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+The doctor frowns. He shakes his head and scribbles something on a page inside a folder
```
Comment (casey-collab) [inline english/call-a-doctor/let-me-out.md]: This file has also been added to a directory that already contained a page.

File: english/call-a-doctor/let-met-out.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+The doctor frowns. He shakes his head and scribbles something on a page inside a folder
```
Comment (casey-collab) [inline english/call-a-doctor/let-met-out.md]: This file has also been added to a directory that already contained a page.

File: english/call-a-doctor/let-met-out.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+The doctor frowns. He shakes his head and scribbles something on a page inside a folder 
+thick with documents. When he looks up, you detect disappointment. The crow's feet 
+scratching at the corners of his eyes tell you that you've made this request before. 
+Maybe hundreds of times.
+
+"I can't do that," the doctor says. "Dr. Sakamoto will be here later on for another round 
+of shock treatments."
+
+[Sit and wait.](../../marshmallow.md)
```
Comment (casey-collab) [inline english/call-a-doctor/let-met-out.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/call-a-doctor/let-met-out.md (hunk 5 above)
Comment (casey-collab) [inline english/call-a-doctor/let-met-out.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

File: english/call-a-doctor/doctor.md (hunk 1 above)
Comment (casey-collab) [inline english/call-a-doctor/doctor.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/call-a-doctor/ghostbusters.md (hunk 2 above)
Comment (casey-collab) [inline english/call-a-doctor/ghostbusters.md]: This file has also been added to a directory that already contained a page.

File: english/call-a-doctor/hungry.md (hunk 3 above)
Comment (casey-collab) [inline english/call-a-doctor/hungry.md]: This file has also been added to a directory that already contained a page.

File: english/call-a-doctor/let-me-out.md (hunk 4 above)
Comment (casey-collab) [inline english/call-a-doctor/let-me-out.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

File: english/call-a-doctor/doctor.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+As you exhale, the walls stiffen, becoming less like marshmallow, and more like 
+the interior of your cell. The walls are a toasted-marshmallow kind of dirty white 
+color, not like the institutionalized white you remember from movies.
+
+The doctor enters.
+
+["Let me outta here, Doc!!!"](let-me-out.md)
```
Comment (casey-collab) [inline english/call-a-doctor/doctor.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/call-a-doctor/doctor.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+
+["I'm uber hungry, Doc."](hungry.md)
```
Comment (casey-collab) [inline english/call-a-doctor/doctor.md]: This link is also invalid.

File: english/call-a-doctor/doctor.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+
+["Is Ghostbusters showing in the common room tonight, Doc?"](ghostbusters.md)
```
Comment (casey-collab) [inline english/call-a-doctor/doctor.md]: This link is also invalid.

File: english/call-a-doctor/ghostbusters/ghostbusters.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+
+"Movie hour is on Tuesday nights, you know that," the doctor explains.
+
+"Ghostbusters! Ghostbusters! Ghostbusters!!!"
+
+The doctor's hands appear with a folder thick with documents. He scribbles on one. 
+Then he looks up at you. You can sense his disappointment. "You can finish the movie 
+next week. Until then ... Dr. Sakamoto will be here later on for another round of shock treatments."
+
+[Sit and wait.](../marshmallow.md)
```
Comment (casey-collab) [inline english/call-a-doctor/ghostbusters/ghostbusters.md]: This link is also invalid.

File: english/call-a-doctor/hungry/hungry.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+
+"Marshmallows, huh?" He scribbles something on a page in a folder thick with
+documents. "Yesterday, you said you were drowning in gravy. The day before that, 
+you thought there was an image of St. Stephen in your meatloaf." The doctor 
+rubs his eyes, radiating disappointment.
+
+"I'm sorry," the doctor says. "Dr. Sakamoto will be here later on for another round 
+of shock treatments."
+
+[Sit and wait.](../marshmallow.md)
```
Comment (casey-collab) [inline english/call-a-doctor/hungry/hungry.md]: This link is also invalid.

File: english/call-a-doctor/let-me-out/let-me-out.md [hunk 12]
```diff
@@ -0,0 +1,9 @@
+The doctor frowns. He shakes his head and scribbles something on a page inside a folder 
+thick with documents. When he looks up, you detect disappointment. The crow's feet 
+scratching at the corners of his eyes tell you that you've made this request before. 
+Maybe hundreds of times.
+
+"I can't do that," the doctor says. "Dr. Sakamoto will be here later on for another round 
+of shock treatments."
+
+[Sit and wait!.](../marshmallow.md)
```
Comment (casey-collab) [inline english/call-a-doctor/let-me-out/let-me-out.md]: This link is also invalid.

File: english/call-a-doctor/hungry/hungry.md [hunk 13]
```diff
@@ -0,0 +1,13 @@
+"Dinner will be served at the standard time," the doctor explains.
```
Comment (casey-collab) [inline english/call-a-doctor/hungry/hungry.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: english/call-a-doctor/let-me-out/let-me-out.md [hunk 14]
```diff
@@ -0,0 +1,9 @@
+The doctor frowns. He shakes his head and scribbles something on a page inside a folder
```
Comment (casey-collab) [inline english/call-a-doctor/let-me-out/let-me-out.md]: This new file also has no link to it.

File: english/call-a-doctor/ghostbusters/ghostbusters.md [hunk 15]
```diff
@@ -0,0 +1,19 @@
+The door opens a crack and a doctor sticks his head in.
```
Comment (casey-collab) [inline english/call-a-doctor/ghostbusters/ghostbusters.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :four_leaf_clover:

**Pull request merged**